lib/utils.js

```diff
@@ -490,6 +490,12 @@ api.verifyCapabilityChain = async ({
       throw error;
     }
 
+    // if root.expires is undefined then Date.parse returns NaN which is falsey
+    const rootExpirationDate = Date.parse(root.expires);
+    if(rootExpirationDate && expiredAfter > rootExpirationDate) {
```

Comment:
I think `expiredAfter` may be a weird name for this parameter. If I pass "March 1" for "expiredAfter", my brain tells me that any zcap should be expired after that date. Which doesn't make any sense; what I'm really passing is "currentDate" or "comparisonDate" something along those lines. This would default to the current date time on the system.

As it stands, my "March 1" `expiredAfter` here could be after a `rootExpirationDate` of "February 28" and this error would be thrown. But it's not "March 1" yet ... nor is it February 28 yet! Clearly, we want to throw because we're *telling* the API that the date to compare against is "March 1" and that date is after February 28, so we *should* throw. But this is not intuitive via the var name.